downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -141,15 +134,25 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
         |}|crypto:PrivateKey|string config?;
     |};
 
-    // previous `jwt:ValidatorSignatureConfig` record
+    // Previous `jwt:ValidatorSignatureConfig` record.
     public type ValidatorSignatureConfig record {|
         // ... other fields
         string|crypto:PublicKey certFile?;
     |};
 
-    // new `jwt:ValidatorSignatureConfig` record
+    // New `jwt:ValidatorSignatureConfig` record.
     public type ValidatorSignatureConfig record {|
         // ... other fields
         string|crypto:PublicKey certFile?;
```

Comment:
Isn't this the same as L140?

Reply:
Fixed with [d36e88eb689b633b5ad1a83b5218d56333650449](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/d36e88eb689b633b5ad1a83b5218d56333650449)